thermal-motion.go

```diff
@@ -39,10 +43,35 @@ type ThermalMotion struct {
 	Verbose          bool   `mapstructure:"verbose"`
 }
 
-func DefaultThermalMotion() ThermalMotion {
+func DefaultThermalMotion(cameraModel string) ThermalMotion {
+	switch cameraModel {
+	case lepton3.Model35:
+		return DefaultLepton35Motion()
+	default:
+		return DefaultLeptonMotion()
+	}
+}
+
+func DefaultLepton35Motion() ThermalMotion {
+	return ThermalMotion{
+		DynamicThreshold: true,
+		TempThresh:       28000, //280 Kelvin ~ 7 degrees
```

Comment:
Is there a reason there is no tempThreshMax here?

Reply:
have rmeove tempthreshmax as we shouldn't need one